trajopt_sco/include/trajopt_sco/optimizers.hpp

```diff
@@ -119,9 +119,154 @@ struct BasicTrustRegionSQPParameters
   bool log_results;     // Log results to file
   std::string log_dir;  // Directory to store log results (Default: /tmp)
 
+  /** @brief If greater than one, multi threaded functions are called */
+  int num_threads;
+
   BasicTrustRegionSQPParameters();
 };
 
+struct BasicTrustRegionSQPUtilFunctions
```

Comment:
What's the purpose of this struct and the threaded version? I don't see that they get used anywhere

Reply:
I switched to the inheritance approach because it allows implementing different threading libraries. For example I will create an implementation that uses the task composer executor so the same thread pool can be used.

Reply:
Those were left over from a previous attempt. They have been removed.